src/main/java/gov/cms/madie/models/measure/MeasureMetaData.java

```diff
@@ -32,4 +32,6 @@ public class MeasureMetaData {
   private String supplementalDataElements;
   private String measureSetTitle;
   private CqlMetaData cqlMetaData;
+
+  private String translatorVersion;
```

Comment:
Functional Acceptance Criteria
1. Verify we have a way to access the current version of the translator in MADiE for each model

Looks like we need the current version of translator, why do we need to assign to any measure ?

Reply:
If required, how do we set the latest versions to measures ? and we may also have to update all measures when a new version of translator is released.

Reply:
Sounds like something was lost in translation to story. We're targeting a way to poll from the UI the currently running versions of each Translator.

Reply:
@RohitKandimalla @jkotanchik-SB So we are not going to store the translator version for a measure in DB?

Reply:
No not tied to a particular measure, It could be an environmental variable pulled from ServiceConfig ? or a new Collection to store the available translator versions ?